src/app/cluster/details/cluster/machine-deployment-list/template.html

```diff
@@ -140,7 +140,7 @@
       <ng-container matColumnDef="actions">
         <th mat-header-cell
             *matHeaderCellDef
-            class="km-header-cell"></th>
+            class="km-header-cell p-5"></th>
```

Comment:
Only thing i noticed was congestion of previous columns

https://user-images.githubusercontent.com/17727069/184307940-9009ac25-6087-4781-9bae-bf0ac2ed0960.mp4